File: Application/WorkWave/Models/Board.cs
using System.ComponentModel.DataAnnotations;

namespace Models
{
    public class Board
    {
        [Key]
        public int ID { get; set; }
        public string? Name { get; set; }
        public string? Description { get; set; }
        public List<Section>? Sections { get; set; } = [];
        public List<User>? Users { get; set; } = [];
    }
}

File: Application/altfront/src/components/Board.js
import React from 'react';
import Section from './Section';

const Board = ({ board }) => {
    return (
        <div className="board">
            <h1>{board.name}</h1>
            <div className="sections">
                <Section sectionName="To Do" boardId={board.id} />
                <Section sectionName="In Progress" boardId={board.id} />
                <Section sectionName="Completed" boardId={board.id} />
            </div>
        </div>
    );
};

export default Board;
